doc/riverctl.1.scd

```diff
@@ -23,14 +23,24 @@ over the Wayland protocol.
 *csd-filter-add* _app-id_
 	Add _app-id_ to the CSD filter list. Views with this _app-id_ are
 	told to use client side decoration instead of the default server
-	side decoration.
+	side decoration. Note that this affects only new views, not already
+	existing ones.
```

Comment:
I think only applying to new views makes sense if we had a command to toggle the CSD mode of a view. That way it would be identical to the float filter.

Reply:
I'd prefer if we kept the behaviour of the different filters consistent.